### PR TITLE
Prompt for a bit more detail in EV1 linear combo construction

### PR DESCRIPTION
**Pull request by StevenClontz**
I'm getting a lot of students who (I think) are guess-checking to a valid combination; this would better prompt them to learn how to use the matrix to extract the combo.

### CODE REVIEW

**Comment by siwelwerd** [comment]
>I'm getting a lot of students who (I think) are guess-checking to a valid combination

I think that's a valid solution strategy, and probably shows more understanding than memorizing something about an RREF matrix (how many of them can explain *why* those numbers in the RREF matrix describe the coefficients needed?)

**Comment by StevenClontz** [comment]
> (how many of them can explain why those numbers in the RREF matrix describe the coefficients needed

Well, I thought I was suggesting we now ask them exactly this!

File: source/linear-algebra/exercises/outcomes/EV/EV1/template.xml [hunk 1]
```diff
@@ -46,8 +46,8 @@ The vector equation has no solutions, so <m>{{v}}</m> is not a linear combinatio
         <knowl>
             <content>
                 <p>
-If your statement was true, describe a linear combination of <m>{{veclist}}</m>
-that equals <m>{{v}}</m>.
+If your statement was true, describe how to use your RREF matrix to construct
```
Comment (StevenClontz) [inline source/linear-algebra/exercises/outcomes/EV/EV1/template.xml:49]:
```suggestion
If your statement was true, explain and demonstrate how to use your RREF matrix to construct
```

**Comment by siwelwerd** [comment]
I am guessing that in response to that prompt you would see a lot of "The numbers in column 4 give the coefficients to construct the the 4th vector as as a linear combination of the first 3" and then verifying that combination.  But that's not explaining at all why those numbers show up in that column.

I think you might have to ask them to write the solution set to get them to that level of reasoning.

Also, whatever change you want to make here needs to be reflected in the sample solution in the appendix.

**Comment by StevenClontz** [comment]
What's your take @jkostiuk ?

**Comment by StevenClontz** [comment]
FWIW my explanation would not yield a full solution set, but instead say "by substituting 0 into the free (non-pivot) variables, we obtain the equations x=X, y=Y, and w=W, and we may verify X[vect]+Y[vect]+0[vect]+W[vect]=[vect]". But of course I'd also be happy with a solution set, then plugging into free variables there.

Being able to guess and check is good, but avoiding cognitive overload each time you need a linear combo from an RREF is also good.

**Comment by jkostiuk** [comment]
I'm largely neutral here.

In my experience using this assessment, I find most students do write out the solution set and then set different variables. Some of them use 0s, but others use 1s, and yet others use something more exotic or an educated guess-and-check. I actually like the variety that students take to the given prompt. 

If I had to vote one way or the other though, I think I'd vote to leave it---because I like that it's providing variety of reasoning as written for my students.

**Comment by StevenClontz** [comment]
Are your students volunteering their reasoning? Because my students are mostly just writing a (usually) valid linear combination without further detail. So, unless I ask, I don't know their thought process.

Is there a way we can prompt better to get some level of detail beyond just writing a valid linear combination, that specifically reminds students to leave some artifact of their reasoning on the page?

**Comment by jkostiuk** [comment]
Ya, they'll say something like "after trial and error I got blah" or "writing out the solutions set and setting a=blah", we get (blah). 

This is largely an artifact of us having established the understanding that we should always share our reasoning in such assessments and then practicing the checkit problems/looking at the template solutions.

**Comment by StevenClontz** [comment]
I'm able to get there with my students, but the conversation is always way easier if I can point to where it specifically asked for the extra detail in the prompt. Hence all the "explain and demonstrates" rather than "what is" questions in the bank.

File: source/linear-algebra/exercises/outcomes/EV/EV1/template.xml [hunk 2]
```diff
@@ -46,8 +46,8 @@ The vector equation has no solutions, so <m>{{v}}</m> is not a linear combinatio
...
+a linear combination of <m>{{veclist}}</m> that equals <m>{{v}}</m>.
```
Comment (StevenClontz) [inline source/linear-algebra/exercises/outcomes/EV/EV1/template.xml:50]:
```suggestion
If your statement was true, explain and demonstrate how to construct
a specific linear combination of <m>{{veclist}}</m> that equals <m>{{v}}</m>.
```

Maybe an okay compromise @jkostiuk @siwelwerd ?

Comment (jkostiuk) [reply source/linear-algebra/exercises/outcomes/EV/EV1/template.xml:50]: I like it!

**Comment by StevenClontz** [comment]
Ah, still not ready until I update the practice exercises

**Comment by StevenClontz** [comment]
I think this has approval so going ahead and self-merging.

**Comment by jkostiuk** [comment]
Oh sorry, I got confused by your previous comment: I thought I was waiting on one more update from you before I was supposed to review/approve.

**Comment by StevenClontz** [comment]
Sorry, I marked it was ready to review but should have commented explicitly.

**Pull request merged**